downloads/swan-lake-release-notes/swan-lake-2201.9.0/RELEASE_NOTE.md

```diff
@@ -262,22 +262,53 @@ To view bug fixes, see the [GitHub milestone for Swan Lake Update 9 (2201.9.0)](
 
 ### Revamped connector updates
 
-The following new connectors along with major updates have been added to the Ballerina library, as part of the Ballerina connector revamp initiative. 
+The following new connectors along with major updates have been added to the Ballerina library, as part of the Ballerina connector revamp initiative.
 All the connectors listed below have been released under new major versions, due to significant changes in the APIs and functionalities.
 
 #### `asana` package
 
 - Enhanced connector APIs by incorporating resource function syntax, along with improved documentation and examples.
 
-#### `candid` package 
+#### `candid` package
 
 - Introduced support for connecting to Candid's Charity Check, Essentials and Premier REST APIs.
 
-#### `github` package 
+#### `confluent.cavroserdes` package
+
+- Introduced support for serializing and deserializing data using Avro schemas stored in the Confluent Schmema Registry.
+
+#### `confluent.cregistry` package
+
+- Introduced support for integrating with the Confluent Schema Registry.
+
+#### `docusign.dsadmin` package
+
+- Enhanced the connector APIs to align with DocuSign Admin APIs by incorporating resource functions.
+- With the new changes the package name is changed to `docusign.dsadmin` which was previously maintained under the name `docusign.admin`.
+- Add the documentation and examples.
```

Comment:
Is this relevant to the release note? There are a few occurrences of this.

Reply:
I also don't think we need to specifically mention about the doc improvements, which is common for all the revamped connectors(in that case we'll end up repeating the same point under every connector).

Reply:
We can mention in the top section that the revamping effort includes documentation and examples improvements including making the documentation and examples consistent across the connectors.

Reply:
> We can mention in the top section that the revamping effort includes documentation and examples improvements including making the documentation and examples consistent across the connectors.

yeah that makes more sense. +1 for the suggestion.